lld/test/ELF/linkerscript/enable-non-contiguous-regions.test

```diff
@@ -0,0 +1,279 @@
+# REQUIRES: x86
+
+# RUN: split-file %s %t
+# RUN: llvm-mc -n -filetype=obj -triple=x86_64 %t/spill.s -o %t/spill.o
+
+## An input section must spill to a later match if the region of its first match
```

Comment:
There are many "must"s in comments. Our convention states what happens and omits "must". Very occasionally we use "should"

Reply:
Done; this is terser than I was able to come up with, which I appreciate.